src/Tools/dotnet-format/dotnet-format.csproj

```diff
@@ -0,0 +1,64 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
```

Comment:
❔  Why not `netcoreapp2.0` like our other projects?

Reply:
Yes, should be `netcoreapp2.0` in dev16.0.x.

Reply:
global tools only support netcoreapp2.1 and above

Reply:
> global tools only support netcoreapp2.1 and above

😕 Which part of it? I would expect anything that supports netcoreapp2.1 to also support netcoreapp2.0.

Reply:
Note: we are on netcoreapp2.1 now in master and netcoreapp2.0 is actually no longer even supported.